certified-connectors/Repfabric Lead Loader/apiDefinition.swagger.json

```diff
@@ -1,14 +1,14 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Lead Loader",
-    "description": "This Connector helps automate the process of creating a \"Lead\" for Repfabric's backend system. It takes data from a csv and invokes multiple calls to the Repfabric system that can create or get the items needed to automate the Lead creation process.",
+    "title": "Repfabric Power Automate Connector",
```

Comment:
@andreroberts1119 the title exceeds the limit of 30 characters, and it contains at two of the restricted words: "Power Automate" and "Connector". May I ask you please to fix and remove the restricted words.

Reply:
We changed the title.